editor/editor_node.cpp

```diff
@@ -491,7 +491,10 @@ void EditorNode::_notification(int p_what) {
 				}
 
 				for (int i = 0; i < addons.size(); i++) {
-					set_addon_plugin_enabled(addons[i], true);
+					if (addons[i].begins_with("res://"))
+						set_addon_plugin_enabled(addons[i], true);
```

Comment:
Just to be sure, there's no case of plugins being enabled with the new system using a relative path? e.g. `addons/myplugin/plugin.cfg` or `../myotherplugin/plugin.cfg`?

Reply:
Yes. Previously, plugins were loaded only by name (before #45316) and this PR is needed only for compatibility with previous project configurations.

Reply:
Yeah I just wondered if the new plugin system would theoretically support relative paths. Because if so, some users might attempt using it this way and will see their paths (under the new system) automatically converted as if they were plugin names under the old system. But maybe that's not a possible use case, in which case the current compat code is fine.

Reply:
No, relative paths are not currently supported by this system.